src/BloomExe/web/ServerBase.cs

```diff
@@ -26,7 +27,7 @@ public abstract class ServerBase : IDisposable
 		/// <summary>
 		/// Listens for requests on "http://localhost:8089/bloom/"
 		/// </summary>
-		private readonly HttpListener _listener;
+		private HttpListener _listener;
```

Comment:
I made this mutable because it can be initiated twice on Windows, if an exception is thrown the first time.